packages/react/src/components/DateTimePicker/DateTimePicker.test.jsx

```diff
@@ -685,6 +694,86 @@ describe('DateTimePicker', () => {
     expect(applyBytton).toBeDisabled();
   });
 
+  it('should disable apply button when absolute TimePickerSpinner input is invalid on the same day', () => {
+    const defaultValues = {
+      timeRangeKind: PICKER_KINDS.ABSOLUTE,
+      timeRangeValue: {
+        startDate: '2020-04-01',
+        startTime: '12:34',
+        endDate: '2020-04-01',
+        endTime: '11:49',
+      },
+    };
+    const wrapper = mount(<DateTimePicker {...dateTimePickerProps} defaultValue={defaultValues} />);
+    jest.runAllTimers();
+    expect(wrapper.find(`.${iotPrefix}--date-time-picker__field`)).toHaveLength(1);
+    expect(wrapper.find(`.${iotPrefix}--date-time-picker__field`).first().text()).toEqual(
+      '2020-04-01 12:34 to 2020-04-01 11:49'
+    );
+
```

Comment:
We can make these test more concrete by checking for the existence of the "Invalid time format." message, so here, for example, we could check:
```jsx
expect(screen.getAllByText('Invalid time format.')).toHaveLength(2);
expect(screen.getByRole('button', {name: 'Apply'})).toBeDisabled();
```